osf/management/commands/create_test_notifications.py

```diff
@@ -1,27 +1,31 @@
 from django.core.management.base import BaseCommand
 from osf.models.notifications import NotificationSubscription
 from osf.models import OSFUser, Node
-from django.utils.crypto import get_random_string
 from django.utils import timezone
 
 class Command(BaseCommand):
-    help = 'Create duplicate notifications for testing'
+    help = 'Create duplicate notifications for testing for a specific node'
 
     def handle(self, *args, **kwargs):
         user = OSFUser.objects.first()
-        node = Node.objects.first()
-        event_name = 'file_added'
+        node = Node.objects.filter(guids___id='3ura2').first()
```

Comment:
Minor: we probably don't actually need this command (to create a dupe in a shell, see second code block), but if so it should probably use `add_arguments` to accept a `Node._id` and dupe an existing notification on that node.

Something like:
```
class Command(BaseCommand):
    help = 'Create duplicate notifications for testing for a specific node'
    
    def add_arguments(self, parser):
        super().add_arguments(parser)
        parser.add_argument(
            '--node',
            type=str,
            required=True,
            help='Node to dupe a notification on',
        )
        
    def handle(self, *args, **kwargs):
        node_id = options.get('node', None)
        # ...
```

```
# Dupe a NotificationSubscription in shell
ns = Node.load(<guid>).notificationsubscriptions.first()
ns.id = None
ns.save() # Counts as a first_save, generates a new `id` and inserts it into the DB
```